app/controllers/games_controller.rb

```diff
@@ -0,0 +1,7 @@
+class GamesController < ApplicationController
+
+  def index
+    @game = Game.first
```

Comment:
In general, we should use the index to list out games.  If we want to show a single one, we should use the show action.  Alternatively we could not load any games and just have static content here.